### ISSUE TITLE
`allowConditional: true` does not error on non-conditional calls

### ISSUE THREAD

**Issue by seanlafferty-ibm** [playwright-community/eslint-plugin-playwright#353]
Hi, these rules have an `allowConditional` option

- https://github.com/playwright-community/eslint-plugin-playwright/tree/main/docs/rules/no-slowed-test.md
- https://github.com/playwright-community/eslint-plugin-playwright/tree/main/docs/rules/no-skipped-test.md

I interpreted the documentation to mean that this:

```json
{
  "playwright/no-skipped-test": [
    "error",
    {
      "allowConditional": true
    }
  ]
}
```

makes `.skip()` disallowed _unless_ a conditional explanation is provided, eg

```ts
test('foo', ({ browserName }) => {
  test.skip(browserName === 'firefox', 'Still working on it') // pass
})

test('foo', ({ browserName }) => {
  test.skip() // fail
})
```

However, the behavior I am seeing is
```ts
test('foo', ({ browserName }) => {
  test.skip(browserName === 'firefox', 'Still working on it') // pass
})

test('foo', ({ browserName }) => {
  test.skip() // pass
})
```

Using `2.2.0`. Do I misunderstand the docs, or is this feature not behaving as intended? Seeing the same behavior for both of the referenced rules.

**Comment by MSroczynski3** [comment]
Great question @seanlafferty-ibm !

I delved into the code of both the rules mentioned by you. 

What `allowConditional` flag does is a bit different than what think. It is combined with `isStandalone` flag to determine if the block with `.skip()` can be accepted. When is the **isStandalone** true?

It is true when `const isStandalone = call.type === 'config'`

And this is defined in https://github.com/playwright-community/eslint-plugin-playwright/blob/main/src/utils/parseFnCall.ts:

```
  if (
    (name === 'test' || name === 'describe') &&
    (node.arguments.length < 2 || !isFunction(node.arguments.at(-1)))
  ) {
    type = 'config'
  }
```

Based on the condition `(node.arguments.length < 2 || !isFunction(node.arguments.at(-1)))`, here are the examples:

1. Considered config (type: 'config'):
```
test.skip(condition, 'message') - 2 args, last is a string (not function)
test.skip() - 0 args (less than 2)
```
2. NOT considered config (type: 'test', 'describe', etc.):
```
test('name', () => {}) - 2 args, last is a function
describe('suite', () => {}) - 2 args, last is a function
test('name', async () => {}) - 2 args, last is an async function
```

**Comment by mskelton** [comment]
@MSroczynski3 I've hidden your answer as off-topic. The issue here is valid and a bug. `allowConditional` should not allow any non-conditional `test.skip()` calls.
